core/core_bind.cpp

```diff
@@ -1478,6 +1478,24 @@ TypedArray<Dictionary> ClassDB::class_get_method_list(const StringName &p_class,
 	return ret;
 }
 
+Variant ClassDB::class_call_static_method(const Variant **p_arguments, int p_argcount, Callable::CallError &r_call_error) {
+	if (p_argcount < 2) {
+		r_call_error.error = Callable::CallError::CALL_ERROR_TOO_FEW_ARGUMENTS;
+		return Variant::NIL;
+	}
+	if (!p_arguments[0]->is_string() || !p_arguments[1]->is_string()) {
+		r_call_error.error = Callable::CallError::CALL_ERROR_INVALID_ARGUMENT;
+		return Variant::NIL;
+	}
+	String p_class, p_method;
+	p_class = p_arguments[0]->stringify();
+	p_method = p_arguments[1]->stringify();
```

Comment:
I'd say these should be `StringName` and should be done like so (and should not be prefixed with `p_`):
```cpp
StringName class_ = *p_arguments[0];
StringName method = *p_arguments[1];
```
As done elsewhere